index.html

```diff
@@ -12,12 +12,23 @@
       name="keywords"
       content="social,social media, instagram, dummygram, clone, project, react,react project, beginner"
     />
+     <script src="https://kit.fontawesome.com/38705f3c43.js" crossorigin="anonymous"></script>
     <title>DummyGram</title>
   </head>
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.jsx"></script>
     <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
     <script src="/src/script.js" type="module"></script>
+    
+    <footer class="white-section" id="footer">
```

Comment:
please don't hard code in html file, we're following reactjs practices and this footer is supposed to be a component and rendered in the client side through react.